### PR TITLE
repaire help on View class document

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

**Comment by dereuromark** [comment]
Wouldn't `$this->viewBuilder()->theme('SuperHot');` suffice as one liner?
People could also place this somewhere else if they want to.

**Comment by saeideng** [comment]
oh , ok good idea

**Comment by ionas** [comment]
I do actually like the example. Gives you an idea where to place it.

**Comment by dereuromark** [comment]
@ionas But that should be in the docs, not the doc block!

**Comment by ionas** [comment]
Agreed!

**Comment by ionas** [comment]
And it is. http://book.cakephp.org/3.0/en/views/themes.html#themes
Is there an option to add this to the class docblock as a link?

File: src/View/View.php [hunk 1]
```diff
@@ -40,7 +40,12 @@
  * layout using `$this->set()`
  *
  * View class supports using plugins as themes. You can set
- * `$this->theme = 'SuperHot'` in your Controller to use plugin `SuperHot` as a
+ * 
+ * `public function beforeRender(\Cake\Event\Event $event)
+ * {
+ *      $this->viewBuilder()->theme('SuperHot');
+ *  }`
```
Comment (markstory) [inline src/View/View.php]: Examples should start with 3 backticks.

**Comment by markstory** [comment]
Short examples in the API are :+1:. We have lots of examples already and more make the API a more useful resource in my opinion.

**Comment by dereuromark** [comment]
@markstory I think documentation can put those examples in better perspective. The API doc block can only do that much without getting out of hand. In this example one could easily add the theme() call inside beforeRender() as well as any other callback prior to that one - or even the action itself.
Thus IMO the doc block should be kept simple. But I dont mind either way.

**Pull request merged**